README.md

```diff
@@ -266,6 +266,41 @@ For an NPM package, `export =` is accurate if `node -p 'require("foo")'` is the
 
 Then you will have to add a comment to the last line of your definition header (after `// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped`): `// TypeScript Version: 2.1`.
 
+#### I want to use features from TypeScript 3.1 or above.
+
+You will need to use the `typesVersions` feature of TypeScript 3.1 and above. You can find a detailed explanation
+of this feature in the [official TypeScript documentation](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-1.html#version-selection-with-typesversions).
+
+Here's a short explanation to get you started:
+
+1. You'll have to add a `package.json` file to your package definition, with the following contents:
+
+```json
+{
+    "private": true,
```

Comment:
I don't think we want this entry

Reply:
Actually, I believe we do. This prevents someone from inadvertently trying to publish the package. Our types publisher ignores this field when it generates the actual package.json file.

Reply:
Yeah, it doesn't work without this field. The linter complains with:
```
package.json should set `"private": true`
```